pyproject.toml

```diff
@@ -265,6 +265,19 @@ unfixable = [
     "E711"  # NoneComparison. Hard to fix b/c numpy has it's own None.
 ]
 
+[tool.flynt]
+exclude= [
```

Comment:
A bit late to this but should we also exclude logger?

https://discuss.python.org/t/safer-logging-methods-for-f-strings-and-new-style-formatting/13802